libs/clients/judicial-system-sp/src/lib/shared.config.ts

```diff
@@ -0,0 +1,45 @@
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import {
+  ConfigType,
+  IdsClientConfig,
+  LazyDuringDevScope,
+  XRoadConfig,
+} from '@island.is/nest/config'
+import { Configuration } from '../../gen/fetch'
+import { JudicialSystemSPClientConfig } from './judicialSystemSPClient.config'
+
+export const SharedApiConfig = {
+  provide: 'JudicialSystemSPApiProviderConfiguration',
+  scope: LazyDuringDevScope,
+  useFactory: (
+    xroadConfig: ConfigType<typeof XRoadConfig>,
+    config: ConfigType<typeof JudicialSystemSPClientConfig>,
+    idsClientConfig: ConfigType<typeof IdsClientConfig>,
+  ) =>
+    new Configuration({
+      fetchApi: createEnhancedFetch({
+        name: 'clients-judicial-system-sp',
+        organizationSlug: 'domsmalaraduneytid',
+        logErrorResponseBody: true,
```

Comment:
We absolutely sure that the error response body coming from them doesn't contain any PII?